ikarus/io/resultevaluators.hh

```diff
@@ -49,9 +43,9 @@ struct VonMises
       const auto s_x  = resultArray(0, 0);
       const auto s_y  = resultArray(1, 0);
       const auto s_z  = resultArray(2, 0);
-      const auto s_yz = resultArray(4, 0);
-      const auto s_xz = resultArray(5, 0);
-      const auto s_xy = resultArray(6, 0);
+      const auto s_yz = resultArray(3, 0);
```

Comment:
Can you indicate in the 2D case above, that this assumes a plane stress state. I think we forgot that back then.